sources/packages/web/src/services/http/ConfigApi.ts

```diff
@@ -20,4 +21,19 @@ export class ConfigApi extends HttpBaseClient {
       throw error;
     }
   }
+
+  public async getUserInfo(): Promise<any> {
+    try {
+      console.dir(AppConfigService.shared.authService?.tokenParsed);
+      const response = await this.apiClient.get('/user-info', {
+        headers: {
+          Authorization: `Bearer ${AppConfigService.shared.authService?.token}`
```

Comment:
Can you please change it to use the method in HttpBaseClient to add the auth header?